content/en/docs/about/contributing.md

```diff
@@ -67,7 +67,7 @@ you've made
 
 When you are ready to join
 
-- Send a PR adding yourself as a member in [org.yaml](https://github.com/kubeflow/internal-acls/blob/master/github-orgs/kubeflow/org.yaml#L19)
+- Send a PR adding yourself as a member in [org.yaml](https://github.com/kubeflow/internal-acls/blob/master/github-orgs/kubeflow/org.yaml#L19) following instructions in [Joining Kubeflow GitHub organization](https://github.com/kubeflow/internal-acls#joining-kubeflow-github-organization)
```

Comment:
Suggested revision:

- Send a PR adding yourself as a member in [org.yaml](https://github.com/kubeflow/internal-acls/blob/master/github-orgs/kubeflow/org.yaml#L19). Follow the instructions in [Joining the Kubeflow GitHub organization](https://github.com/kubeflow/internal-acls#joining-kubeflow-github-organization) for more information about artifacts to include with your PR and how to test your PR.